app/main.ipynb

```diff
@@ -100,18 +102,33 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "import numpy as np\n",
+    "\n",
+    "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
+    "\n",
+    "subsequence = array[[1, 2, 3, 4, 5]]\n",
```

Comment:
The subsequence selection is incorrect. You should select the first, third, and last elements of the array as per the task description.